app/index.html

```diff
@@ -82,6 +82,7 @@
     <script src="bower_components/angular-bootstrap/ui-bootstrap-tpls.js"></script>
     <script src="bower_components/lodash/lodash.js"></script>
     <script src="bower_components/bootstrap/dist/js/bootstrap.js"></script>
+    <script src="bower_components/bootstrap-combobox/js/bootstrap-combobox.js"></script>
```

Comment:
@spadgett, I think we probably need to revisit all of PatternFly's dependencies then as there are quite a few I don't think we are using but are also included here.